lib/build.api.js

```diff
@@ -11,10 +11,9 @@ module.exports = function (Aquifer) {
   /**
    * Creates a build object from which operations can be initiated.
    *
-   * @param object project object of project that should be built.
    * @param string destination path to destination folder.
    */
-  var Build = function(project, destination) {
+  var Build = function(destination) {
```

Comment:
@mattgrill like, `Path = require('path');`? Standardizing on capitalizing libraries? I'm down for that.